src/components/UserInfo.js

```diff
@@ -1,14 +1,16 @@
 export default class UserInfo {
-    constructor(nameSelector, professionSelector) {
+    constructor(nameSelector, professionSelector, imageSelector) {
         this._name = document.querySelector(nameSelector);
         this._profession = document.querySelector(professionSelector);
+        this._image = document.querySelector(imageSelector);
     }
 
     // Retrieve profile info from the page
     getUserInfo() {
         return {
             name: this._name.textContent,
             profession: this._profession.textContent,
+            avatar: this._image.src
```

Comment:
You should not return the avatar here, since it's for the profile form. You can create a separate method `getAvatar` if you need that

Reply:
Sure thing! That's fixed. Thank you.